lib/fund.js

```diff
@@ -32,9 +32,19 @@ class Fund extends BaseCommand {
     return 'fund'
   }
 
+  /* istanbul ignore next - see test/lib/load-all-commands.js */
+  static get params () {
+    return [
+      'json',
+      'browser',
+      'unicode',
+      'which',
+    ]
+  }
+
   /* istanbul ignore next - see test/lib/load-all-commands.js */
   static get usage () {
-    return ['[--json] [--browser] [--unicode] [[<@scope>/]<pkg> [--which=<fundingSourceNumber>]']
+    return ['[<@scope>/]<pkg>]']
```

Comment:
```suggestion
    return ['[[<@scope>/]<pkg>]']
```

Reply:
Hm, some information is lost here, because the previous usage communicated that the `--which` configuration value is only relevant if a package name is specified, but I don't think that's a big deal, personally.

Reply:
Or even better, maybe we should just stop doing `[[@<scope>/]<pkg>]` everywhere, and replace it with just `[<pkg>]`.

Package names can have scopes.  Everyone knows this now.  No need to keep being noisy about it.

Reply:
Yeah the information is lost here, but that's spelled out in the extended help everywhere. This is just telling you what params are available.